Google.Cloud.EntityFrameworkCore.Spanner.Samples/Snippets/ReadOnlyTransactionSample.cs

```diff
@@ -49,16 +49,14 @@ public static async Task Run(string connectionString)
 
         // Create a new database context and insert a singer with the given id. This singer will not be visible
         // to the read-only transaction.
-        using (var writeContext = new SpannerSampleDbContext(connectionString))
```

Comment:
This indented context was intentional, as it should not be used (and should not be accessible) after this block.

Reply:
Done!!